config/config.yaml

```diff
@@ -34,6 +34,12 @@ assets:
     i18next:
         basePath: '%assets_static_relative%/i18next/dist/umd/'
         script: i18next.min.js
+    i18next-xhr-backend:
+        basePath: '%assets_static_relative%/i18next-xhr-backend/dist/umd/'
+        script: i18nextXHRBackend.min.js
+    i18next-browser-languagedetector:
+        basePath: '%assets_static_relative%/i18next-browser-languagedetector/dist/umd/'
+        script: i18nextBrowserLanguageDetector.min.js
```

Comment:
note these were separated out since these 2 are not mission critical (the ordering of libraries brought in are how they are ordered in this file).
i18next is mission critical (to feed the super duper js xl function created by super duper @sjpadgett ) and needs to be ordered highly in ordering when bringing in libraries, whereas these other 2 are not (just used in dicom and do not need to brought in a high order)

Reply:
> super duper js xl function created by super duper @sjpadgett

Ya shouldn't be so modest @bradymiller! That was a team effort between Matrix, you and me. The function doesn't appear like much on first glance but, I still think it was one of our better collaborations, well openemr profiling is up there too.